src/components/SearchResultTile/index.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { render, screen } from "@testing-library/react";
+
+import SearchResultTile from "./index";
+
+describe("Search Problem Result Tile", () => {
```

Comment:
Usually I'd go with:
```suggestion
describe("<SearchResultTile />", () => {
```